superset/translations/pt_BR/LC_MESSAGES/messages.po

```diff
@@ -2075,238 +2075,238 @@ msgstr ""
+msgstr "Legenda"
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:1113
 msgid "Whether to display the legend (toggles)"
 msgstr "Se deseja exibir a legenda (alternar)"
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:1118
 msgid "Show Values"
-msgstr "Mostrar valores"
+msgstr "Mostrar Valores"
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:1121
 msgid "Whether to display the numerical values within the cells"
 msgstr "Se deseja exibir os valores numéricos dentro das células"
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:1126
 msgid "X bounds"
-msgstr "X limites"
+msgstr "Limitis do Eixo X"
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:1129
 msgid "Whether to display the min and max values of the X axis"
 msgstr "Se deseja exibir os valores mínimo e máximo do eixo X"
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:1134
 msgid "Y bounds"
-msgstr "Y limites"
+msgstr "Limitis do Eixo Y"
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:1137
 msgid "Whether to display the min and max values of the Y axis"
 msgstr "Se deseja exibir os valores mínimo e máximo do eixo Y"
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:1142
 msgid "Rich Tooltip"
-msgstr "Rich Tooltip"
+msgstr "Ferramenta de +Informações"
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:1145
 msgid "The rich tooltip shows a list of all series for that point in time"
 msgstr ""
-"A ferramenta de informação rica mostra uma lista de todas as séries para "
+"A ferramenta de +Informações mostra uma lista de todas as séries e valores para "
 "esse ponto no tempo"
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:1151
 msgid "Y Log Scale"
-msgstr "Y Escala de registro"
+msgstr "Escala Log. - Eixo Y"
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:1154
 msgid "Use a log scale for the Y axis"
-msgstr "Use uma escala de log para o eixo Y"
+msgstr "Utilizar uma escala logarítmica para o eixo Y"
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:1159
 msgid "X Log Scale"
-msgstr "X Log Scale"
+msgstr "Escala Log. - Eixo X"
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:1162
 msgid "Use a log scale for the X axis"
-msgstr "Use uma escala de log para o eixo X"
+msgstr "Utilizar uma escala logarítmica para o eixo X"
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:1167
 msgid "Log Scale"
 msgstr "Escala logarítmica"
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:1170
 msgid "Use a log scale"
-msgstr "Use uma escala de registro"
+msgstr "Utilizar uma escala de logarítmica"
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:1175
 msgid "Donut"
-msgstr "Rosquinha"
+msgstr "Rosca"
 
 #: superset/assets/javascripts/explore/stores/controls.jsx:1178
 msgid "Do you want a donut or a pie?"
-msgstr "Você quer uma rosquinha ou uma torta?"
+msgstr "Você quer um gráfico de rosca ou de pizza?"
```

Comment:
On json, you used "torta" for pie chart now your using "pizza"

Reply:
O json foi construido a partir do .po, conforme instruido aqui no git do superset. Provavelmente devido a atenção acabei deixando passar as duas formas. Irei efetuar a correção

Reply:
Talked to a Brazilian friend and he pointed out that on Brazil it's more common to say "Gráfico de pizza" yet, I see a lot of "Gráfico de torta" still, can you change it just to make it more coherent